### PR TITLE
Adds up metric, stops exiting on network failure

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Fixes https://github.com/giantswarm/prometheus-kemp-exporter/issues/15

We have had issues where network failure causes the exporter
to exit, and this then requires an operator to restart
the service. Now, we set a metric that a failure has occured,
which we can alert on.

### CODE REVIEW

File: cmd/server.go [hunk 1]
```diff
@@ -157,8 +171,13 @@ func serverRun(cmd *cobra.Command, args []string) {
 			virtualServices, err := client.ListVirtualServices()
 			if err != nil {
 				log.Println("Error getting virtual services ", err)
-				os.Exit(1)
+				kempUp.Set(0)
+
+				sleep()
+				continue
 			}
+			kempUp.Set(1)
```
Comment (JosephSalisbury) [inline cmd/server.go]: Slight difference here, in that we do two network calls, so I'm setting after both of them.

**Comment by JosephSalisbury** [comment]
@xh3b4sd RFR

**Comment by JosephSalisbury** [comment]
A framework for this and the pingdom exporter is not a bad idea, now that i think about it

**Comment by xh3b4sd** [comment]
Ok, LGTM

**Pull request merged**